chromiumoxide_types/src/lib.rs

```diff
@@ -209,6 +209,17 @@ pub enum Message<T = CdpJsonEventMessage> {
     Event(T),
 }
 
+impl<'de, T>  Deserialize<'de> for Message<T> where T: for<'t> Deserialize<'t> {
+    fn deserialize<D>(deserializer: D) -> Result<Self, D::Error> where D: Deserializer<'de> {
+        let value = Value::deserialize(deserializer)?;
+        if let Ok(response) = serde_json::from_value(value.clone()) {
+            return Ok(Self::Response(response))
+        }
+        use serde::de::{Error};
+        Ok(Self::Event(serde_json::from_value(value.clone()).map_err(|err|Error::custom(format!("{:?}", err)))?))
```

Comment:
hmm, how is this different from untagged deserialization?

Reply:
CdpEventMessage deserialization doesn't play nice with untagged enums, i don't remember exactly why, but this adds the necessary coersion for it to work.

Reply:
maybe it was fixed in the meantime somehow, let me try on master for a bit

Reply:
it doesn't seem to be happening anymore, but longer tests are necessary